src/ignitus-StudentSignUp/Components/index.js

```diff
@@ -1,2 +1,2 @@
 // eslint-disable-next-line import/prefer-default-export
-export { default as Signup } from './Signup';
+export { default as Signup } from '../profile/Components/profile';
```

Comment:
Rename the `profile.js` file to `SignUp.js` because profile and SignUp do not correlate to each other. It doesn't sound good in terms of the code structuring.